src/components/ToastSnackbar.tsx

```diff
@@ -102,20 +42,8 @@ export default function ToastSnackbarMessage(): JSX.Element {
       id='snackbar'
       className={classes.mainSnackbar}
     >
-      <div className={`${classes.mainContainer} ${classes.toast} body${snackbar.priority}`}>
-        <div className={`${classes.iconContainer} iconContainer`}>
-          <Icon name={snackbar.priority} className='snackbarIcon' />
-        </div>
-        <div className={`${classes.body} body`}>
-          {snackbar.title && (
-            <Typography component='p' variant='body1' className={classes.snackbarTitle}>
-              {snackbar.title}
-            </Typography>
-          )}
-          <Typography component='p' variant='body1'>
-            {snackbar.msg}
-          </Typography>
-        </div>
+      <div className={classes.toastContainer}>
```

Comment:
div is needed to avoid mui error and give full width for toast to adjust width internally
```
index.js:1 Warning: Failed prop type: Invalid prop `children` supplied to `ForwardRef(Grow)`. Expected an element that can hold a ref. Did you accidentally use a plain function component for an element instead? For more information see https://mui.com/r/caveat-with-refs-guide
    at Grow (http://localhost:3000/static/js/vendors~main.chunk.js:319610:5)
    at Snackbar (http://localhost:3000/static/js/vendors~main.chunk.js:335948:86)
```